services/ingestion/etl/just/csv/serverless.yml

```diff
@@ -0,0 +1,78 @@
+service: ingestion-etl-just-csv
+
+plugins:
+  - serverless-webpack
+
+custom:
+  webpackIncludeModules: true
+  eubfrEnvironment: ${opt:eubfr_env, file(../../../../../config.json):eubfr_env, env:EUBFR_ENV, 'dev'}
+  bucketName: ${file(../../../../../resources/harmonized-storage/serverless.yml):custom.bucketName}
+
+package:
+  individually: true
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+  timeout: 60
+  stage: ${opt:stage, file(../../../../../config.json):stage, 'dev'}
+  region: ${opt:region, file(../../../../../config.json):region, 'eu-central-1'}
+  deploymentBucket:
+    name: eubfr-${self:custom.eubfrEnvironment}-deploy
+  stackTags:
+    ENV: ${self:custom.eubfrEnvironment}
+  iamRoleStatements:
+    -  Effect: "Allow"
+       Action:
+         - "s3:PutObject"
+       Resource:
+         Fn::Join:
+           - ""
+           - - "arn:aws:s3:::"
+             - ${self:custom.bucketName}
+             - "/*"
+
+functions:
+  parseCsv:
+    handler: src/events/onParseCSV.handler
+    name: ${self:provider.stage}-${self:service}-parseCsv
+    memorySize: 256
+    environment:
+      BUCKET: ${self:custom.bucketName}
+      REGION: ${self:provider.region}
+      STAGE: ${self:provider.stage}
+    events:
+      - sns:
+          arn:
+            Fn::Join:
+              - ""
+              - - "arn:aws:sns:"
+                - Ref: "AWS::Region"
+                - ":"
+                - Ref: "AWS::AccountId"
+                - ":${self:provider.stage}-etl-just-csv"
+          topicName: ${self:provider.stage}-etl-just-csv
+
+resources:
+  Resources:
+    SNSTopicBUDGCSV:
+      Type: AWS::SNS::Topic
+      Properties:
+        TopicName: ${self:provider.stage}-etl-just-csv
+        DisplayName: BUDG CSV ETL
+    SNSTopicBUDGCSVPolicy:
+      Type: AWS::SNS::TopicPolicy
+      Properties:
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            -
+              Sid: AllowIngestionManagerToPushNotificationEffect
```

Comment:
That's actually `AllowAnyoneToPingSNSTopicJUSTCSV`, so that other services can communicate to https://github.com/ec-europa/eubfr-data-lake/pull/123/files#diff-7c3f3b1c92473646f28385bce14cf1b6R78 that something has happened.

Could you please also check whether the service this line is taken from also need update of this?